CLN: Removed unreached else block GH33478

FollowTheProcess opened the pull request:
Unreached else block in method _wrap_applied_output was removed.

Local tests passed before and after change

- [ ] xref #33478 (Not fully closed, this addresses 1 part)
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

Commit:
CLN: Removed unreached else block GH33478

Unreached else block in method _wrap_applied_output was removed.

Local tests passed before and after change.

MarcoGorelli commented on pandas/core/groupby/generic.py:
```diff
@@ -1373,10 +1373,6 @@ def _wrap_applied_output(self, keys, values, not_indexed_same=False):
                 # of columns
                 return self.obj._constructor_sliced(values, index=key_index)
 
-        else:
```
Thanks @FollowTheProcess - while you're here, can you change the `elif` in line 1221 to `else`?

FollowTheProcess replied:
Sorry yep, will do this and the xref now

FollowTheProcess replied:
That's now done :) Thanks for helping me contribute (in an albeit microscopic way) the contributing docs made it super easy as a newbie. I hope to be able to contribute more as I learn more.

MarcoGorelli commented:
> closes #33478 (Not fully closed, this addresses 1 part)

Can you change "closes" to "xref"?

Commit:
CLN GH33478 Correct elif to else in _wrap_applied_output.

Related to previous removal of unreached else block.
Removal left an out of place elif, replaced with else.

Local tests pass before/after

MarcoGorelli commented:
The `Windows py37_np18` shows a failure (`pandas/tests/base/test_unique.py::test_unique_bad_unicode[idx_or_series_w_bad_unicode0]`), although if I check out your pull request and run it myself it passes (also, none of the functions in `pandas/tests/base/test_unique.py` hit this code path), so am pretty sure the CI failure is unrelated

FollowTheProcess commented:
Oh, weird! I ran the full test suite (test_fast.sh) before committing and no failures for me either. Let me know if you need me to do anything else

jreback commented:
thanks @FollowTheProcess

FollowTheProcess commented:
Thanks both for the help. I hope to contribute more in the future!
